Sources/Extensions/UIKit/UIStackViewExtensions.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  UIStackViewExtensions.swift
+//  SwifterSwift-iOS
+//
+//  Created by Benjamin Meyer on 2/18/18.
+//
```

Comment:
Hey @BennX Could you please add the copyright //  Copyright © 2018 SwifterSwift 
The same way the other files :)

Reply:
Sure. Somehow I thought the Template did that already.